processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -153,6 +153,12 @@ func setHostCPUInfo(d *Detector, cpuInfo cpu.InfoStat) error {
 	}
 	d.rb.SetHostCPUFamily(family)
 
+	// For windows, this field is left blank. See https://github.com/shirou/gopsutil/blob/v3.23.9/cpu/cpu_windows.go#L113
+	// Set the model to "0" to avoid parsing bug.
+	// ISSUE: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27675
+	if cpuInfo.Model == "" {
+		cpuInfo.Model = "0"
```

Comment:
Thank's for dealing with this! 

Maybe we should just skip setting this instead of using the zero value?

Reply:
That also makes sense.
Will change the commit to skip this setting.

Reply:
Makes sense to me to skip rather than using 0